modules/silicon_design/scripts/build/images/provision/environment.yml

```diff
@@ -34,10 +34,55 @@ dependencies:
   - pandas
   - pymeep=*=mpi_mpich_*
   - jupyterlab
-  - python
+  - litex-hub::gcc-riscv32-elf-newlib
+  - litex-hub::openfpgaloader
+  - litex-hub::dfu-util
+  - litex-hub::flterm
+  - litex-hub::openocd
+  - litex-hub::verilator
+  - litex-hub::nextpnr-nexus
+  - litex-hub::nextpnr-ecp5
+  - litex-hub::nextpnr-ice40
+  - litex-hub::yosys
+  - litex-hub::iceprog
+  - litex-hub::prjxray-tools
+  - litex-hub::prjxray-db
+  - litex-hub::vtr-optimized
+  - litex-hub::symbiflow-yosys-plugins
+  - lxml
+  - simplejson
+  - intervaltree
+  - json-c
+  - libevent
+  - python=3.7
   - pip
   - pip:
     - klayout
     - scrapbook[gcs]
     - google-cloud-aiplatform
     - cloudml-hypertune
+    - simplejson
+    - termcolor
+    - tqdm
+    - yapf==0.24.0
+    - requests
+    - meson
+    - Pillow
+    - intervaltree
+    - junit-xml
+    - numpy
```

Comment:
any reason we don't install the conda version of those?

Reply:
will move these to above pip